filament/src/ResourceAllocator.cpp

```diff
@@ -162,6 +162,12 @@ backend::TextureHandle ResourceAllocator::createTexture(const char* name,
         uint32_t width, uint32_t height, uint32_t depth,
         std::array<backend::TextureSwizzle, 4> swizzle,
         backend::TextureUsage usage) noexcept {
+    // If this texture is a color attachment, then make sure it can be the source of a copy for
+    // readPixels().
+    if (any(usage & backend::TextureUsage::COLOR_ATTACHMENT)) {
+        usage |= backend::TextureUsage::BLIT_SRC;
+    }
+
```

Comment:
we shouldn't do this here. for the framegraph, we should use the correct usage bits. also I don't think any of these textures are ever used with readPixels(), but if they are, the pass doing that needs to call `read()` with the proper usage bit.

Reply:
moved to the right place (on the picking texture).